tcms/testplans/static/testplans/js/get.js

```diff
@@ -302,3 +351,22 @@ function sortTestCases() {
     //put the new order in the DOM
     tcsParentElement.html(visibleTCrows);
 }
+
+function getSelectedTestCases() {
```

Comment:
Similar to selectedCheckboxes() in testruns/js/get.js on the other refactoring branch. Add a TODO comment on top so we don't forget to evaluate the 2 functions later and remove one of them if possible.

Reply:
I checked it out but didnt find much similarities. I will add the comment.